src/calculateRentalCost.js

```diff
@@ -5,6 +5,21 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_FOR_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const result = PRICE_FOR_DAY * days;
+
+  if (days >= LONG_TERM) {
+    return result - LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    return result - SHORT_TERM_DISCOUNT;
+  }
```

Comment:
Consider removing the `else` keyword as it is unnecessary after a return statement. The code after the `if` block won't be executed if the condition is true, so you can simply use another `if` statement. This will make the code cleaner and easier to read. Refer to the checklist item 5 for an example of the preferred structure.